CHANGES.md

```diff
@@ -13483,7 +13446,7 @@ ndif
    instead.  BIO_gethostbyname() does not know what timeouts are
    appropriate, so entries would stay in cache even when they have
    become invalid.
-   *Bodo Moeller; problem pointed out by Rich Salz <rsalz@zolera.com>
+   *Bodo Moeller; problem pointed out by Rich Salz <rsalz@zolera.com>*
```

Comment:
@richsalz you worked for a car rental company?

Reply:
> you worked for a car rental company?

No, it was a startup that tried to provide a digital signature signing device.  It failed. :)